cli/src/pcluster/models/cluster.py

```diff
@@ -663,28 +645,47 @@ def start(self):
         except Exception as e:
             raise _cluster_error_mapper(e, f"Failed when starting compute fleet with error: {str(e)}")
 
+    def start_slurm_compute_fleet(self):
```

Comment:
The start function was split to two to solve `function too complex` complain by linters